content/en/quickstart/device-to-device.md

```diff
@@ -11,3 +11,24 @@ menu:
     weight: 20
 toc: true
 ---
+
+## Initialize CoAP(over UDP) Client
+
+## Discovery Cloud Server example 
+
+## On-board Cloud Server example 
+
+## Provision Cloud Configuration 
+### Provision Access Control for Cloud 
```

Comment:
This is too detailed for quick start. We can than add it to development section, but here it's enough to discover, transfer ownership, interact with device on D2D level - proximity network, provision cloud config, disown.

So sections could be like
`Discover device on local network`
`Transfer ownership`
`Interact with the device locally`

(cloud provisioning will be on another page, this is d2d).

To each section add also 1-2 sentences of description, answering what and why.

Reply:
@ondrejtomcik 
Ok I got it.